gwcs/tests/test_wcs.py

```diff
@@ -1894,3 +1895,13 @@ def test_parameterless_transform():
 
     assert gwcs.invert(1, 1) == (1, 1)
     assert gwcs.invert(1 * u.pix, 1 * u.pix) == (1, 1)
+
+
+def test_fitswcs_imaging():
+    projection = Pix2Sky_TAN()
+    crval = [5.6, -72.4]
+    crpix = [5, 5]
+    fwcs = fitswcs.FITSImagingWCSTransform(projection, crpix=crpix, crval=crval)
+    ra, dec = fwcs(*crpix)
+    assert_allclose((ra, dec), crval)
+    assert_allclose(fwcs.inverse(ra, dec), crpix)
```

Comment:
This looks like a test of the transform model not a gwcs. could you wrap it into a WCS object. Maybe include that wcs in the examples module.